docs/installation/docker-compose.md

```diff
@@ -64,51 +59,61 @@ There is also a legacy project called [Docker Toolbox](https://docs.docker.com/t
 
     **Swap**: Swap space is space borrowed from your hard disk drive to serve as makeshift RAM as needed. If you cannot provide as much RAM as you would like, increase this as is reasonable given your free disk space.
 
-## Launching Islandora with Docker
-
-**Summary:** Using git, clone the isle-dc project. In that directory, enter the command `make demo` to build the docker infrastructure and a demo site.  Then use the command `docker-compose up -d` to start the containers.
-
-!!! hint "ISLE-DC variants: `make demo` vs `make local`"
-    The `make demo` command alone will spin up a sandbox-like version of ISLE, but only with a front end. The code files will be inaccessible, and you will not be able to install additional modules or themes. The `make local` command creates a full-fleged development environment. It will copy the active Drupal codebase locally in a way that is also live to the ISLE site. This method takes longer (and may require multiple retries if your internet connection is spotty) but is required if you will be testing pull requests or writing code. The `make` command alone will build the infrastructure but not install anything (including Drupal!).
+### Getting isle-dc
 
 ```bash
 git clone https://github.com/islandora-devops/isle-dc
 cd isle-dc
-make demo
 ```
 
+## Deploying Islandora 
+
+From within the `isle-dc` directory, you will have `make` comands at your disposal for building and maintaining your repository. Configuration is contained in a `.env` file.
```

Comment:
So all the make commands below (in the Useful Make Commands section) work on Demo too?